app/views/pages/home.html.haml

```diff
@@ -40,9 +40,8 @@
           %th= @weekly_bookings.count
           %td people booked within past week
         %tr.current-week-released
-          - total = @weekly_bookings.count
-          - inactive = @weekly_bookings.inactive.count
-          %th #{inactive} (#{present_percent(inactive, @weekly_bookings.count)})
+          - released = @weekly_releases.count
+          %th #{released} (#{present_percent(released, @active_bookings.count)})
```

Comment:
To use an extreme illustration, if there are two inmates and one is released, the release percentage would show that 100% of inmates were released.